app/importers/trade_lead/fbopen_importer.rb

```diff
@@ -0,0 +1,13 @@
+# For shared methods used by the two FBO importers
+module TradeLead
+  module FbopenImporter
+    def process_geo_fields(entry)
+      entry[:country] = IsoCountryCodes.find(entry[:country]).alpha2 rescue nil
+      entry.merge! add_geo_fields([entry[:country]])
+    end
+
+    def extract_end_date(entry)
+      [entry[:arch_date], entry[:resp_date]].reject(&:nil?).max
+    end
+  end
+end
```

Comment:
I think you better give this module a different name. You've given it the same name as the module that contains the classes that you include it into (see trade_lead/fbopen_importer/full_data.rb below). I'm not sure why your code is actually working with that in place, but the phrase "circular dependencies" is already entering my head :smile: